exporter/clickhouselogsexporter/factory.go

```diff
@@ -33,7 +33,7 @@ const (
 	archiveNamespace = "clickhouselogs-archive"
 	databaseName     = "signoz_logs"
 	tableName        = "logs"
-	migrationsFolder = "./migrations"
+	migrationsFolder = "/logsmigrations"
```

Comment:
Running binary directly instead of docker image will not work this change?

Reply:
In case of running directly, one can set the environment variable:

`export LOG_MIGRATIONS_FOLDER="./logsmigrations"`

For most case that is running in Docker/K8s, this newly introduced changes should work fine.
Also, keeping it consistent with the traces migration constant.

Reply:
It works fine in docker/k8s, but that isn't my original concern. Generally, programs have defaults, and env/cli args are added to support the customisation. The stand-alone binary should run with defaults without any custom configuration, but this change doesn't make it work.

Reply:
yeah, true. It's been set to `/migrations` as default for traces.
That should be updated.